### PR TITLE
[CI:DOCS] Update how to enable resource limit delegation

### PR DESCRIPTION
**Pull request by sstosh**
Add a information about `cpu-shares` option and `CPUSET` limits.

Signed-off-by: Toshiki Sonoda <sonoda.toshiki@fujitsu.com>

#### Does this PR introduce a user-facing change?

```release-note
Update how to enable resource limit delegation
```

### CODE REVIEW

**Review by flouthoc — APPROVED** [review]
LGTM

File: troubleshooting.md [hunk 1]
```diff
@@ -704,19 +709,19 @@ Example output might be:
 
     memory pids
 
-In the above example, `cpu` is not listed, which means the current user does
-not have permission to set CPU limits.
+In the above example, `cpu` and `cpuset` is not listed, which means the current user does
```
Comment (rhatdan) [inline troubleshooting.md:712]: Please change to: 
```
In the above example, `cpu` and `cpuset` are not listed, which means the current user does
not have permission to set CPU or CPUSET limits.
```

File: troubleshooting.md [hunk 2]
```diff
@@ -704,19 +709,19 @@ Example output might be:
...
+not have permission to set CPU and CPUSET limits.
 
-If you want to enable CPU limit delegation for all users, you can create the
+If you want to enable CPU and CPUSET limit delegation for all users, you can create the
```
Comment (rhatdan) [inline troubleshooting.md:715]: Nit: CPU or CPUSET

**Comment by rhatdan** [comment]
Thanks @sstosh 
A couple of small English cleanups.

**Comment by rhatdan** [comment]
Thanks @sstosh 
/lgtm

**Pull request merged**